index.js

```diff
@@ -29,7 +42,38 @@ async function run() {
     if (!containerDef) {
       throw new Error('Invalid task definition: Could not find container definition with matching name');
     }
-    containerDef.image = imageURI;
+
+    // Check for imageURI
+    if(imageURI) {
```

Comment:
since neither input is strictly required now, can we add some debug statements to indicate which course(s) of action the workflow is taking, and throw an error if _neither_ is provided? That way if I wanted to see what was happening I could see:

(image specified) -------> "inserting container image [IMAGE]" 
(merge file specified) ---> "merging task definition fragment [FILE_NAME]"
(nothing specified) -----> ERROR: "No image or merge file specified. Please specify one or both to use this action."